samples/nextjs/scripts/strip.js

```diff
@@ -0,0 +1,89 @@
+const fs = require('fs');
```

Comment:
I like it! I think we should productize this functionality by:

1. Moving to base package (`sitecore-jss-cli` with `create`? or maybe `sitecore-jss-dev-tools`?)
2. Allowing start/end flag name as a parameter. This way, you could have multiple conditional/stripped features at once.